### PR TITLE
Remove comma-dangle check

### PR DESCRIPTION
**Pull request by Somotaw**

### CODE REVIEW

**Comment by TooAngel** [comment]
DCBOT: 
This repository is under [democratic collaboration](https://github.com/TooAngel/democratic-collaboration) and will be merged automatically.

The merge decision is based on the outcome of the reviews:
 - `Approve` add the reviewer value (number of commits) to the `metric`
 - `Request changes` substract the reviewer value from the `metric`

 - `metric > 99%` merge now
 - `metric > 75%` merge in 1 day
 - `metric > 50%` merge in 3 days
 - `metric >= 0%` merge in 7 days

Please review the PR to make a good democratic decision.

This is the placeholder for reviewer summoning :-) (top two + random)

**Comment by TooAngel** [comment]
DCBOT: Current status percentage: 0 votes: 34 total: 260

**Comment by TooAngel** [comment]
DCBOT: Current status percentage: 0 votes: 181 total: 260

**Comment by TooAngel** [comment]
DCBOT: Current status percentage: 0 votes: 181 total: 260 age: 0

**Comment by TooAngel** [comment]
DCBOT: Current status coefficient: 0 votes: 181 total: 260 age: 0

**Comment by TooAngel** [comment]
DCBOT: Current status coefficient: 0 votes: 228 total: 260 age: 0

File: .codeclimate.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ engines:
       - javascript
   eslint:
     enabled: true
+    checks:
+      comma-dangle:
+        enabled: false
```
Comment (Nevrdid) [inline .codeclimate.yml]: Approved too 👍

**Comment by TooAngel** [comment]
DCBOT: Current status coefficient: 0 votes: 228 total: 260 age: 0

**Comment by TooAngel** [comment]
DCBOT: Current status coefficient: 0 votes: 228 total: 260 age: 0

**Comment by TooAngel** [comment]
DCBOT: Current status coefficient: 0 votes: 228 total: 260 age: 0

**Comment by TooAngel** [comment]
DCBOT: Current status coefficient: 0 votes: 228 total: 260 age: 0

**Comment by TooAngel** [comment]
DCBOT: Current status coefficient: 0.876923076923 votes: 228 total: 260 age: 0

**Comment by Somotaw** [comment]
@mschultheiss83

**Comment by Somotaw** [comment]
@aceofaces 
@Tsungen 
@Evanito

**Comment by TooAngel** [comment]
DCBOT: Current status coefficient: 0.9 votes: 234 total: 260 age: 0

**Comment by TooAngel** [comment]
DCBOT: Current status coefficient: 0.907692307692 votes: 236 total: 260 age: 0

**Comment by TooAngel** [comment]
\o/

**Pull request merged**